src/App.jsx

```diff
@@ -32,11 +32,11 @@ function App() {
 
   return (
     <div className="h-100 vh-100 w-100 d-flex flex-column" style={{minHeight: '100%'}}>
-      <TabSwitch activeTab={activeTab} setActiveTab={setActiveTab}>
-        {(data !== null && map !== null) ? <WorldMap data={data} map={map} isActiveTab={activeTab === 0} icon="" title="Single Category Map"/>: <pre>"Loading..."</pre>}
-        {(data !== null && map !== null) ? <CountryDistance data={data} map={map} isActiveTab={activeTab === 1} icon="" title="Difference Map"/> : <pre>"Loading..."</pre>}
+      <TabSwitch activeTabNumber={activeTabNumber} setActiveTabNumber={setActiveTabNumber}>
+        {(data !== null && map !== null) ? <WorldMap data={data} map={map} isActiveTab={activeTabNumber === 0 ? true : undefined} icon="" title="Single Category Map"/>: <pre>"Loading..."</pre>}
+        {(data !== null && map !== null) ? <CountryDistance data={data} map={map} isActiveTab={activeTabNumber === 1 ? true : undefined} icon="" title="Difference Map"/> : <pre>"Loading..."</pre>}
```

Comment:
Yeah, it isn't pretty. I did it because I got a react warning at some point, and after googling it I found that other people got rid of the warning by doing it this way.

Tried to switch it back to "activeTabNumber === 0" again, and the warnings are still gone :O So I guess I can change it back then

Reply:
Don't have to, I was just wondering